man/systemd-nspawn.xml

```diff
@@ -719,6 +719,15 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>--syscall-whitelist</option></term>
```

Comment:
we tend to suffix options that accept arguments with "=" everywhere, i.e. please write this as `<option>--syscall-whitelist=</option>`